### PR TITLE
rename uop [run_process_replay] 

### PR DESCRIPTION
**Pull request by DKormann**
not changing extra.

renaming UPat as well otherwise there are collisions

### CODE REVIEW

**Comment by geohot** [comment]
btw, process replay should work on this

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -43,9 +43,9 @@ repos:
         language: system
         always_run: true
         pass_filenames: false
-      - id: pylint
-        name: pylint
-        entry: env PYTHONPATH="." python3 -m pylint tinygrad/
-        language: system
-        always_run: true
-        pass_filenames: false
+      # - id: pylint
```
Comment (Qazalin) [inline .pre-commit-config.yaml:46]: ?

File: tinygrad/renderer/cstyle.py [hunk 2]
```diff
@@ -104,10 +104,10 @@ def ssa(prefix:str, u:Optional[UOp]=None):
       c[prefix] += 1
       return ret
 
-    child_count = Counter(v for ru in uops for v in ru.vin)
+    child_count = Counter(v for ru in uops for v in ru.src)
 
     for u in uops:
-      uop,dtype,vin,args = u.uop,u.dtype,u.vin,u.arg
+      uop,dtype,vin,args = u.op,u.dtype,u.src,u.arg
```
Comment (Qazalin) [inline tinygrad/renderer/cstyle.py:110]: why is cstyle sitll using uop and vin

Comment (geohot) [reply tinygrad/renderer/cstyle.py:110]: I think renaming that is a bigger change than it needs to be. The smaller you can make PRs the better.

**Comment by DKormann** [comment]
would make new pr for vin-src

**Pull request merged**